user.go

```diff
@@ -100,3 +100,16 @@ func (c *Client) DeleteUser(userID string) error {
 func (c *Client) DeleteUserWithContext(ctx context.Context, userID string) error {
 	return c.del(ctx, "users/"+userID)
 }
+
+// RetrieveMe retrieve the current user profile.
```

Comment:
```suggestion
// RetrieveMe retrieves the current user.
```

Reply:
This wasn't completely resolved - these aren't really "profiles", but I won't block on that either.

Reply:
My bad I will address this in the next PR.